stories/atoms/Card.stories.tsx

```diff
@@ -1,29 +1,48 @@
 import { Meta, Story } from '@storybook/react';
 import React from 'react';
-import { Heading } from '../../src';
-import { Card, CardProps } from '../../src';
+import { Card, CardProps, SimpleGrid } from '../../src';
 
 export default {
   title: 'Components/Card',
   component: Card,
 } as Meta;
 
+const lorem =
+  'Lorem ipsum dolor sit amet consectetur adipisicing elit. Officia dignissimos ut voluptate eveniet? Sunt totam, quidem, nobis corrupti voluptatum aliquam expedita voluptas alias laborum eos dolor esse fugiat? Explicabo, tenetur.';
+
 const Template: Story<CardProps> = (args) => <Card {...args} />;
 
-export const WithHeading = Template.bind({});
-WithHeading.args = {
-  children: (
-    <Heading
-      variant="texturina"
-      textTransform="uppercase"
-      fontSize={{ base: '16px' }}
-      mb={5}
-      textAlign="center"
-    >
-      Card Title
-    </Heading>
-  ),
+export const Single = Template.bind({});
+Single.args = {
+  title: 'Card Title',
+  text: lorem,
+  imageSrc:
+    'https://res.cloudinary.com/saimano/image/upload/v1622036153/RaidGuild/icons/red/consultations_lr6ef4.png',
 };
 
-export const Empty = Template.bind({});
-Empty.args = {};
+const images = [
+  'https://res.cloudinary.com/saimano/image/upload/v1622036153/RaidGuild/icons/red/consultations_lr6ef4.png',
```

Comment:
Same with these -- perhaps converting them to data:uri embedded prevents us from having to worry about these going missing if the source ever goes down.